lib/g_searcher_web/controllers/search_result_controller.ex

```diff
@@ -9,7 +9,7 @@ defmodule GSearcherWeb.SearchResultController do
 
     search_results = SearchResults.list_search_results_by_user_id(user_id, params["query"])
 
-    render(conn, "index.html", search_results: search_results)
+    render(conn, "index.html", search_results: Enum.with_index(search_results, 1))
```

Comment:
I think we can move `Enum.with_index/2`  to the template, it feels like a view logic.